code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -553,18 +570,46 @@
 	name = "Curse of Madness"
 	desc = "Curses the station, warping the minds of everyone inside, causing lasting traumas. Warning: this spell can affect you if not cast from a safe distance."
 	cost = 4
+	ritual_invocation = "ALADAL DESINARI ODORI'IN PORES ENHIDO'LEN MORI MAKA TU"
 
 /datum/spellbook_entry/summon/curse_of_madness/Buy(mob/living/carbon/human/user, obj/item/spellbook/book)
 	SSblackbox.record_feedback("tally", "wizard_spell_learned", 1, name)
 	active = TRUE
 	var/message = stripped_input(user, "Whisper a secret truth to drive your victims to madness.", "Whispers of Madness")
 	if(!message)
-		return FALSE
```

Comment:
Maybe just have this loop until message has contents. It could've been an accident.

Reply:
Doing accidents in ritual tab is just being imcopetent enougg, and it shouldn't be `return` or your wiz point can be used for another magic after multiplied.

Reply:
Actually nvm. I got what you mean. Loop until you get message